### PR TITLE
Fix VB legacy project system interop definition

### PR DESCRIPTION
**Pull request by jasonmalinowski**

### CODE REVIEW

**Review by CyrusNajmabadi — APPROVED** [review]

**Review by ryzngard — APPROVED** [review]

**Review by ToddGrun — APPROVED** [review]
:shipit:

**Review by jaredpar — APPROVED** [review]

File: src/VisualStudio/VisualBasic/Impl/ProjectSystemShim/Interop/VBCompilerOptions.vb [hunk 1]
```diff
@@ -131,8 +131,7 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic.ProjectSystemShim.
         ''' <summary>
         ''' Preferred load address.
         ''' </summary>
-        ''' <remarks></remarks>
-        Public dwLoadAddress As UInteger
+        Public dwLoadAddress As UIntPtr
```
Comment (jaredpar) [inline src/VisualStudio/VisualBasic/Impl/ProjectSystemShim/Interop/VBCompilerOptions.vb:134]: Our hungarian notation no longer matches the type

Comment (jasonmalinowski) [reply src/VisualStudio/VisualBasic/Impl/ProjectSystemShim/Interop/VBCompilerOptions.vb:134]: @jaredpar: this matches the internal interop definition. We'll keep it consistent.

Comment (jaredpar) [reply src/VisualStudio/VisualBasic/Impl/ProjectSystemShim/Interop/VBCompilerOptions.vb:134]: Mostly playing around here ;)

**Pull request merged**